src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static final Map<String, Integer> fruits = new HashMap<>();
+    private static final int INITIAL_QUANTITY = 0;
+
+    public static void addFruit(String fruit, int quantity) {
+        fruits.put(fruit, fruits.getOrDefault(fruit, INITIAL_QUANTITY) + quantity);
+    }
+
+    public static void removeFruit(String fruit, int quantity) {
+        if (fruits.getOrDefault(fruit, INITIAL_QUANTITY) < quantity) {
+            throw new RuntimeException("Not enough " + fruit + " in storage");
+        }
+        fruits.put(fruit, fruits.get(fruit) - quantity);
+    }
+
+    public static Map<String, Integer> getAllFruits() {
+        return new HashMap<>(fruits);
+    }
+}

```

Comment:
The `Storage` class exposes its internal data structure through the `getAllFruits` method. This could lead to unwanted modifications from outside the class. Consider returning an unmodifiable view of the map using `Collections.unmodifiableMap(fruits)` to ensure encapsulation.